frontend/src/services/errorConstants.ts

```diff
@@ -0,0 +1,31 @@
+export enum ErrorConstants {
+  VALIDATION_ERROR = "VALIDATION_ERROR",
+  UNIQUE_ERROR = "UNIQUE_ERROR",
+  END_OF_REGISTRATION_ERROR = "END_OF_REGISTRATION_ERROR",
+  BAD_REQUEST_ERROR = "BAD_REQUEST_ERROR",
+  ACCESS_DENIED_ERROR = "ACCESS_DENIED_ERROR",
+  RESOURCE_ALREADY_EXISTS_ERROR = "RESOURCE_ALREADY_EXISTS_ERROR",
+  EXISTING_DEPENDENCY_ERROR = "EXISTING_DEPENDENCY_ERROR",
+  NOT_FOUND_ERROR = "NOT_FOUND_ERROR",
```

Comment:
Check with backend for all errors

Reply:
Add `NOT_DELETABLE_ERROR`

Exchange `END_OF_REGISTRATION_ERROR` with `WRONG_TIME_EXCEPTION`